cypress/integration/data-import/mrc-import-with-matching-001.spec.js

```diff
@@ -0,0 +1,140 @@
+import SettingsDataImport from '../../support/fragments/data_import/settingsDataImport';
+import dataImport from '../../support/fragments/data_import/dataImport';
+import jobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import TopMenu from '../../support/fragments/topMenu';
+import MatchProfiles from '../../support/fragments/data_import/match_profiles/match-profiles';
+import logs from '../../support/fragments/data_import/logs';
+import inventorySearch from '../../support/fragments/inventory/inventorySearch';
+import SearchInventory from '../../support/fragments/data_import/searchInventory';
+import getRandomPostfix from '../../support/utils/stringTools';
+import NewFieldMappingProfile from '../../support/fragments/data_import/mapping_profiles/newMappingProfile';
+import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewActionProfile from '../../support/fragments/data_import/action_profiles/newActionProfile';
+import ActionProfiles from '../../support/fragments/data_import/action_profiles/actionProfiles';
+import NewJobProfile from '../../support/fragments/data_import/job_profiles/newJobProfile';
+import exportFile from '../../support/fragments/data-export/exportFile';
+import ExportMarcFile from '../../support/fragments/data-export/export-marc-file';
+import FileManager from '../../support/utils/fileManager';
+
+describe('ui-data-import: Test MARC-MARC matching for 001 field', () => {
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    cy.visit(TopMenu.dataImportPath);
+  });
+
+  it('C17044: MARC-MARC matching for 001 field', () => {
```

Comment:
add tag related with test type. See example in https://wiki.folio.org/display/DQA/Cypress#Cypress-Cypress-grepplugin